antiope-sns/src/Antiope/SNS.hs

```diff
@@ -10,13 +10,12 @@ import Data.Text       (Text)
 import Network.AWS     (MonadAWS, send)
 import Network.AWS.SNS
 
-publishMsg :: MonadAWS m => Text -> Text -> m ()
+publishMsg :: MonadAWS m => Text -> Text -> m (Maybe Text)
```

Comment:
These `Text` parameters are a bit confusing... Fortunately, we are in a strongly typed environment where types are  cheap! :)))

Can we newtype these things?
```
newtype TopicArn = TopicArn Text deriving (Show, Eq, Ord, ToText, FromText)
newtype MessageId = MessageId Text deriving (Show, Eq, Ord, ToText, FromText)
```

and then 

```
publishMsg :: MonadAWS m => TopicArn -> Text -> m (Maybe MessageId)
```

Reply:
I would also suggest not to shortcut `Message` for `Msg`. 
Why not calling it `publishMessage`? We  already have `ackMessage`, `sendMessage`, why have we started doing `msg` in other places?

Reply:
I suppose amazonka provides types for these as well, maybe?

Reply:
* I think we can use `Topic` and `Endpoint` types
* Use `Msg` suffix is because the name shadowing, as `publish` is used by amazonka.sns. Or we can use other names like `sendNotif`, `registerNotif` ...
* Will change to `publishMessage`.